oe_theme.theme

```diff
@@ -116,3 +116,46 @@ function oe_theme_preprocess_menu_local_tasks(&$variables) {
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_input__radio().
+ */
+function oe_theme_preprocess_input__radio(&$variables) {
+  _oe_theme_preprocess_input_label_wrapper($variables);
+}
+
+/**
+ * Implements hook_preprocess_input__checkbox().
+ */
+function oe_theme_preprocess_input__checkbox(&$variables) {
```

Comment:
If the label processing is the same for all form elements, can't we use `oe_theme_preprocess_input(&$variables)` ?

Reply:
We need to move the label element to the input template only for checkbox and radio elements. Truth is that the name `_oe_theme_preprocess_input_label_wrapper()` is not the best to represent this common function, but I couldn't come with anything better to improve it.

Reply:
The function `oe_theme_preprocess_input__[something](&$variables)` is a variant of `oe_theme_preprocess_input(&$variables)`, this is why I was wondering if all the inputs had to be updated.
If it was the case, just naming the function `oe_theme_preprocess_input(&$variables)` would have been enough to alter them all.
It's also a good news that Drupal is using variant in theme calls, it's actually a very good news.

Regarding the name of the common function, I don't have a better candidate, I've always been very bad choosing arbitrary names :)